Article/Article.js

```diff
@@ -3,18 +3,18 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelectorAll('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
     
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.forEach((item) => item.textContent = 'expand');
```

Comment:
You're missing this piece to get your functionality to work, let me know if you need some help finishing this project up. Also, you're trying to merge to Lambda's repo and not your own so I wouldn't be able to merge your request had it been working. Just a heads up.